server/schema/donations.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE status AS ENUM ('pending', 'approved', 'denied', 'flagged', 'scheduling', 'scheduled', 'picked up', 'failed');
+
+
+CREATE TABLE [IF NOT EXISTS] donations (
+   id SERIAL PRIMARY KEY,
+   route_id INT,
+   order_num INT,
+   status status NOT NULL,
+   address_street VARCHAR(256) NOT NULL,
+   address_unit VARCHAR(256),
+   address_city VARCHAR(256) NOT NULL,
+   address_zip int NOT NULL,
+   first_name VARCHAR(256) NOT NULL,
+   last_name VARCHAR(256) NOT NULL,
+   email VARCHAR(256) NOT NULL,
+   phone_num VARCHAR(15) NOT NULL,
+   notes VARCHAR(256),
+   date date
+);
```

Comment:
Have you guys ran this create statement in pgAdmin? Remember to drop the old table since you've made changes to the schema so the new table will get created.

Reply:
Let's add the `DROP TABLE` and `DROP TYPE` statements to have them ready if we need to run these queries.